package.json

```diff
@@ -13,15 +13,21 @@
   "devDependencies": {
     "@algolia/client-search": "4.9.1",
     "@babel/cli": "7.13.0",
+    "@babel/plugin-transform-react-jsx": "^7.13.12",
     "@babel/preset-env": "7.13.8",
     "@babel/preset-react": "7.12.13",
     "@babel/preset-typescript": "7.13.0",
+    "@rollup/plugin-json": "4.1.0",
+    "@rollup/plugin-node-resolve": "7.1.1",
+    "@rollup/plugin-replace": "2.3.3",
     "@testing-library/react": "11.2.6",
     "@types/jest": "^26.0.23",
     "@types/react": "^17.0.5",
     "@typescript-eslint/eslint-plugin": "2.19.0",
     "@typescript-eslint/parser": "2.19.0",
     "babel-eslint": "10.1.0",
+    "babel-plugin-module-resolver": "^4.1.0",
```

Comment:
This dep should be pinned.

Reply:
Done in https://github.com/algolia/react-recommendations/pull/6/commits/47398ad5fd3e21ff8e9edad6b2a0ea0eeb4b9444